doc/nrf/gs_programming.rst

```diff
@@ -87,4 +87,78 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
    	In a multi-image build, this allows you to debug the source code of your application only.
 
 If you get an error that a tool or command cannot be found, first make sure that the tool is installed.
-If it is installed, verify that its location is correct in the PATH variable or, if applicable, in the SES settings.
\ No newline at end of file
+If it is installed, verify that its location is correct in the PATH variable or, if applicable, in the SES settings.
+
+.. _gs_programming_cmd:
+
+Building with Command line
+**************************
+
+Complete the following steps to build |NCS| projects with command line after completing the :ref:`command line build environment setup for Windows<build_environment_cli_win>` or the :ref:`command line build environment setup for Linux<build_environment_cli_linux>` for Windows and Linux environments respectively.
+
+1.    Open a terminal window.
+
+#.    Go to the samples or applications directory depending on whether you are building a sample or an application:
+
+      .. code-block:: console
+
+         cd nrf/samples
+         cd nrf/applications
+
+      .. note::
+         All the samples corresponding to the nRF9160 DK board are located at :file:`samples/nrf9160`.
+
+#.    Build the sample or application using the west command.
+      The development board is specified by the parameter ``board_name``, and the sample or application directory is specified by the parameter ``directory_name`` in the west command as follows:
+
+      .. code-block:: console
+
+         west build -b board_name directory_name
+
+      See `Board names <Board names_>`_ for more information on the development boards.
+      To reuse an existing build directory for building another sample or application for another board, pass ``-p=auto`` to ``west build``.
+
+      After running the ``west build`` command, the build files are generated in ``build/zephyr``.
+      For more information on the contents of the build directory, see `Zephyr documentation on Build Directory Contents`_.
+
+      .. include:: gs_programming.rst
+         :start-after: .. imp_note_nrf91_start
+         :end-before: .. imp_note_nrf91_end
+
+#.    Connect the development board to your PC using a USB cable.
+#.    Power on the development board.
+#.    Program the sample or application to the board from the build directory using the following command:
+
+      .. code-block:: console
+
+         west flash
+
+#.    Reset the board to start the sample or application.
+
+For more information on building and programming using command line, see the Zephyr documentation on :ref:`zephyr:west-build-flash-debug`.
+
+.. _gs_programming_board_names:
+
+Board names
+***********
+
+.. _table:
+
++-------------------+---------------------------------------------------+
+| Development Boards| Board names                                       |
++===================+===================================================+
```

Comment:
I suggest to use the official kit names here (as we have them in the shortcuts.txt file), and link to the respective Zephyr pages. So for example :ref:`nRF51 DK board (PCA10028)<zephyr:nrf51_pca10028>`.

Reply:
Modified.